src/components/ButtonPanel.js

```diff
@@ -9,13 +9,19 @@ const ButtonPanel = () => {
     ['1', '2', '3', '+'],
     ['0', '.', '='],
   ];
-
+  const size = sign => sign === '0' && true;
+  const changeColor = (ind, length) => (length === ind + 1 ? 'color' : '');
```

Comment:
Please don't abbr `ind` write the complete word so it will be easier to read and understand.

I would suggest you use a actual color here instead of the string `color`, see my previous comments on this.
Also, to make the default prop work, you will need to use `undefined` or `null` for when `length === ind + 1` is false.

Reply:
I'll make corrections on this.